lib/functoria/cli.ml

```diff
@@ -34,20 +34,24 @@ let description_section = "DESCRIBE OPTIONS"
 type query_kind =
   [ `Name
   | `Packages
-  | `Opam
-  | `Install
-  | `Files of [ `Configure | `Build ]
+  | `Opam of [ `Global | `Local ]
+  | `Files
+  | `Dune of [ `Config | `Build | `Project | `Workspace | `Dist ]
   | `Makefile ]
 
 let query_kinds : (string * query_kind) list =
   [
     ("name", `Name);
     ("packages", `Packages);
-    ("opam", `Opam);
-    ("install", `Install);
-    ("files-configure", `Files `Configure);
-    ("files-build", `Files `Build);
+    ("local.opam", `Opam `Local);
```

Comment:
instead of `local` / `global` maybe name these with the intended goal? `build` / `release`  ?

Reply:
The thing is you can have "build" tools such as `crunch` or ppx's that are locally fetched in your unikernel workspace, so that might be confusing. The technical requirements are:
- `local`: builds with dune (either runtime libraries or build-time tools)
- `global`: only build-time tools